message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -48,6 +48,16 @@
         <param index="6">Empty.</param>
         <param index="7">Empty.</param>
       </entry>
+      <entry value="217" name="MAV_CMD_DO_SCRIPTING" hasLocation="false" isDestination="false">
```

Comment:
Perhaps 
```suggestion
      <entry value="217" name="MAV_CMD_DO_SEND_SCRIPT_MESSAGE" hasLocation="false" isDestination="false">
```

It bothers me this is broadcast.  But I guess we've no way to distinguish scripts.